compiler/src/iree/compiler/InputConversion/MHLO/ConvertMHLOToLinalgExt.cpp

```diff
@@ -410,7 +410,7 @@ struct ReverseOpConversion : public OpConversionPattern<mhlo::ReverseOp> {
     Value emptyTensor =
         rewriter.create<tensor::EmptyOp>(loc, mixedSizes, ty.getElementType());
     rewriter.replaceOpWithNewOp<IREE::LinalgExt::ReverseOp>(
-        op, op->getResultTypes(), adaptor.getOperands(), emptyTensor,
+        op, adaptor.getOperands().getType(), adaptor.getOperands(), emptyTensor,
```

Comment:
Nit: Using operand types works, but really you want the `init` types I think? In this case it happens to be the same

Reply:
I changed to using the type convert as its the technically correct version.